Tests/StryngTests/StryngTests.swift

```diff
@@ -42,6 +42,11 @@ class StryngTests: XCTestCase {
         let example = "Example"
         XCTAssertNil(example[0...8])
         XCTAssertNil(example[10...11])
+        
```

Comment:
Can you please split into separate tests these with negative indices?

Reply:
Sure, Done :)